docs/builtins.md

```diff
@@ -1560,14 +1560,14 @@ does not change. The initial color is `"black"`.
 
 ### `hsl`
 
-`hsl` changes the color of the pen, similar to [`color`](#color). However, the
-arguments to the `hsl` function are provided as numbers and therefore are
-more suited to mathematical manipulation, e.g. to find the complimentary
-color, create a trail in animations with the alpha value or create color
-gradients.
-
-`hsl` uses the CSS (Cascading Style Sheets) [hsl function] to specify the
-color. In Evy the `hsl` function takes one to four arguments: `hue`,
+`hsl` returns a string to be used with [`color`](#color) and[`clear`]
+(#clear) functions. The arguments to the `hsl` function are numbers and
+therefore are more suited to mathematical manipulation, e.g. to find the
+complimentary color, create a trail in animations with the alpha value or
+create color gradients.
+
+`hsl` returns an CSS (Cascading Style Sheets) [hsl function] string specifying
+a color. In Evy the `hsl` function takes one to four arguments: `hue`,
 `saturation`, `lightness`, and `alpha`.
```

Comment:
Perhaps reword this. I was going to say get rid of "In Evy", but I think you should describe the parameters first perhaps:

`hsl` takes one to four arguments: ... and returns a [CSS hsl function] string specifiying a color.

Note I've changed the anchor text to include "CSS"